source/community/release_notes/quince.rst

```diff
@@ -15,26 +13,156 @@ These are the release notes for the Quince release, the 17th community release o
 Breaking Changes
 ****************
 
+* `Include and enable the Indigo theme in the default Open edX image <https://github.com/overhangio/tutor/issues/953>`__
+  * Deployers running OpenEdX with the default theme and no additional plugins will have the Indigo theme enabled automatically.
+* `Deployers must ensure that for all of their IDAs, any JWT_PUBLIC_SIGNING_JWK_SET Django setting does not contain whitespace inside of the Base64 strings of the encoded keys <https://github.com/openedx/edx-drf-extensions/blob/master/CHANGELOG.rst#880---2023-05-16>`__
+* Django-storages upgraded to latest version and it has some breaking changes.
+  * The constructor kwarg bucket is no longer accepted. Instead, use bucket_name.
+  * define default_acl value explicitly in constructor kwarg e.g default_acl: public-read. in previous versions django-storages provides default value as public-read but now it is none. So it's important to mention it explicitly as per your use case.
+* For Django 4.2 CSRF_TRUSTED_ORIGINS must include scheme. Update all urls there with schemes. e.g .edx.org changes to ``https://*.edx.org``.
+* Docker requirements have changed: They are now: Docker v24.0.5+, with BuildKit 0.11+. 
 
 Learner Experiences
 *******************
 
+* `feat: fetch program subscription details <https://github.com/openedx/edx-platform/pull/32023>`__
+* `In Unit Discussion Sidebar for Learners <https://partners.edx.org/announcements/discussion-sidebar-release>`__
```

Comment:
partners.edx.org isn't a public site, so we can't link folks to it. 

I suggest:

```
* A new Sidebar enhances the current discussion provider by enabling learners to interact with forums using a collapsible sidebar that accompanies course units. This replaces the use of discussion xblocks.

New course runs and re-runs created after you upgrade to quince will automatically use this new provider. However, existing course runs will not be affected.

To learn more about this upgrade, visit the `wiki page <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3470655498/Discussi...](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3470655498/Discussions+upgrade+Sidebar+and+new+topic+structure>`__.
```

(please double-check my RST)